src/include/ipc/header.h

```diff
@@ -86,10 +86,12 @@
  *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
  *     | Core                                                          |
  *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- *     | Reserved                                                      |
+ *     | ext_data_length                                               |
```

Comment:
Thanks @keyonjie , this is good! There's a note about which version this example applies to above ("ABI 3.17.0 example"). We can fix this in a later update as well, but if you need to respin, then.

Reply:
Thanks, let me respin it.

Reply:
Or actually wait, according to ABI classifier UUID is assigned to 3.17 so this is correct after all! https://github.com/orgs/thesofproject/projects/2 
Sorry for the noise.